lib/node/NodeSourcePlugin.js

```diff
@@ -37,33 +38,39 @@ NodeSourcePlugin.prototype.apply = function(compiler) {
 			if(localOptions.process) {
 				var processType = localOptions.process;
 				parser.plugin("expression process", function() {
-					return ModuleParserHelpers.addParsedVariable(this, "process", "require(" + JSON.stringify(getPathToModule("process", processType)) + ")");
+					return ModuleParserHelpers.addParsedVariable(this, "process", "require(" +
+					    JSON.stringify(path.relative(this.state.module.context, getPathToModule("process", processType))) + ")");
```

Comment:
Didn't noticed that, thank you for point it out. I'll work on it tonight.
BTW, should I keep other changes that cause test errors?
I can eliminate few of them by modifying dependent packages, but it seems that changing webpack codebase to fit these change is beyond my current ability. T_T